doc/hadfs-exe.1

```diff
@@ -0,0 +1,87 @@
+.TH HADFS-EXE 1 "hadfs" "Active Directory File System"
+.SH NAME
+hadfs-exe \- mount AD configuration from LDAP as a FUSE tree
+.SH SYNOPSIS
+.B hadfs-exe
+\fBmountpoint\fR \fBhost\fR [\fBport\fR]
+.SH DESCRIPTION
+.PP
+.B hadfs-exe
+is a software for mapping \fBActive Directory\fR settings stored in
+\fBLDAP\fR database using \fBFUSE\fR (Filesystem in USErspace).
+.PP
+In order for \fBhadfs-exe\fR to be able to get the configuration, it
+must be authorized to do it via Kerberos (using \fBkinit\fR for
+example).
+.TP
+\fBmountpoint\fR \- user-owned directory to mount filesystem
+.TP
+\fBhost\fR \- hostname of LDAP database
+.TP
+\fBport\fR \- LDAP port (default: \fI389\fR)
+.SH FILES
+LDAP tree elements are represented by directories and their properties
+are represented by text files. You may encounter the following files:
+.TP
+\fB.chpwd\fR - Write-only file for changing user's password.
+.TP
+\fB.attributes\fR - Object attributes as a \fBLDIF\fR file.
+.TP
+\fB.attributes.json\fR - Object attributes as a \fBJSON\fR.
+.TP
+\fB.lasterrror\fR - File stored in catalog where the program was
+started. There is no way to get human-readable error message from the
+file system so this file stores text of last error message.
+.TP
+\fB.refresh\fR - The LDAP tree data is cached locally. You may trigger
+cache update by 'touch .refresh' command invoked in the directory you
+want to update.
+.SH EXAMPLES
+.PP
+Mount Active Directory domain settings into user-owned directory in
+\fI$HOME/ad-settings\fR:
+.PP
+.EX
+$ \fBmkdir\fR /home/user/ad-settings
+$ \fBkinit\fR administrator@DOMAIN.ALT
+$ \fBhadfs-exe\fR /home/user/ad-settings dc0.domain.alt
+.EE
+.SH SEE ALSO
+.BR kinit (1),
+.BR <https://www.altlinux.org/ActiveDirectory/ADFS>
+.SH AUTHOR
+This software is written by Sergei Bubnov.
+.SH REPORTING BUGS
+Report bugs to: <https://github.com/omgbebebe/hadfs>
```

Comment:
maybe it should point to github.com/altlinuxteam/hasdf?

Reply:
Fixed.